dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/EnrollmentAnalyticsQueryCriteria.java

```diff
@@ -77,6 +77,8 @@
 
     private Integer pageSize;
 
+    private Boolean paging = true;
```

Comment:
I think we can use primitive boolean here to avoid null checks/autoboxing, also to make it consistent with `EventsAnalyticsQueryCriteria` where we use primitive boolean paging.

Reply:
Done @larshelge